### PR TITLE
Improving CI and including E2E to run

### PR DESCRIPTION
**Pull request by donpui**
Adding E2E tests without large files testing, setting to run one instance at the time. 
Separated wdio.CI.conf.ts to have dedicated for CI.
Additionally tried to added HTML reporter, but it failed to run on Github.
E2E will run only on Pull Requests, not on each push.

### CODE REVIEW

File: client-e2e/test/specs/receive.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ describe("Receive flow", () => {
         await (await Page.receiveButton()).click();
         const input = await Page.receiveCodeInput();
         await input.click();
-        await browser.keys(["7-guitarist-revenge"]);
+        await browser.keys(["100-guitarist-revenge"]);
```
Comment (meejah) [inline client-e2e/test/specs/receive.ts:36]: why 100 instead of 7?

Comment (donpui) [reply client-e2e/test/specs/receive.ts:36]: Thought, if run against real mailbox service (w.*, w3.*), theoretically might interrupt sender and test result might be different. Higher number, less probability.

Comment (meejah) [reply client-e2e/test/specs/receive.ts:36]: Then use a nameplate that can't be automatically allocated (and a comment to this effect would be nice).

Comment (donpui) [reply client-e2e/test/specs/receive.ts:36]: Maximum nameplate number is 999? 1000 should be good

Comment (meejah) [reply client-e2e/test/specs/receive.ts:36]: yes

**Comment by donpui** [comment]
Fails to build here, I will open separate PR

**Pull request closed without merging**